fastlane/lib/fastlane/actions/device_grid/README.md

```diff
@@ -6,6 +6,8 @@ No more manually installing and testing your app just to review a PR.
 
 ![assets/GridExampleScreenshot.png](assets/GridExampleScreenshot.png)
 
+[Open Example Pull Request](https://github.com/Themoji/ios/pull/12#issuecomment-215836315)
```

Comment:
Oh I just mean it's a link. Something like `Click here to view an example pull request`. What would you recommend wording wise?